<?xml version="1.0"?>

<launch>
<node name="onboard_logging" pkg="vigir_ocs_logging" type="logger.py" output="screen">
    <param name="enable_Log_grabbing" type="bool" value="true" />
    <param name="logging_location" type="str" value="/home/vigir/Experiments" />
    <param name="logger_name" type="str" value="example_" />
    <rosparam param="to_log">[ '/trajectory_controllers/right_arm_traj_controller/state',
      '/trajectory_controllers/right_arm_traj_controller/command',
      '/trajectory_controllers/right_arm_traj_controller/follow_joint_trajectory/goal',
      '/trajectory_controllers/right_arm_traj_controller/follow_joint_trajectory/feedback',
      '/trajectory_controllers/right_arm_traj_controller/follow_joint_trajectory/result',
      '/trajectory_controllers/left_arm_traj_controller/state',
      '/trajectory_controllers/left_arm_traj_controller/command',
      '/trajectory_controllers/left_arm_traj_controller/follow_joint_trajectory/goal',
      '/trajectory_controllers/left_arm_traj_controller/follow_joint_trajectory/feedback',
      '/trajectory_controllers/left_arm_traj_controller/follow_joint_trajectory/result',
      '/joint_controllers/right_arm_joint_limit_traj_controller/follow_joint_trajectory/goal',
      '/joint_controllers/right_arm_joint_limit_traj_controller/follow_joint_trajectory/cancel',
      '/joint_controllers/right_arm_joint_limit_traj_controller/follow_joint_trajectory/feedback',
      '/joint_controllers/right_arm_joint_limit_traj_controller/follow_joint_trajectory/result',
      '/joint_controllers/left_arm_joint_limit_traj_controller/follow_joint_trajectory/goal',
      '/joint_controllers/left_arm_joint_limit_traj_controller/follow_joint_trajectory/cancel',
      '/joint_controllers/left_arm_joint_limit_traj_controller/follow_joint_trajectory/feedback',
      '/joint_controllers/left_arm_joint_limit_traj_controller/follow_joint_trajectory/result',
      '/robot_controllers/footstep_controller/execute_step_plan/cancel',
      '/robot_controllers/footstep_controller/execute_step_plan/feedback',
      '/robot_controllers/footstep_controller/execute_step_plan/goal',
      '/robot_controllers/footstep_controller/execute_step_plan/result',
      '/flor/controller/walk_command',
      '/flor/controller/step_command',
      '/flor/controller/walk_feedback',
      '/flor/controller/step_feedback',
      '/robot_controllers/footstep_controller/execute_step_plan/cancel',
      '/mode_controllers/control_mode_controller/change_control_mode/cancel',
      '/mode_controllers/control_mode_controller/change_control_mode/feedback',
      '/mode_controllers/control_mode_controller/change_control_mode/goal',
      '/mode_controllers/control_mode_controller/change_control_mode/result',
      '/vigir_move_group/cancel',
      '/vigir_move_group/feedback',
      '/vigir_move_group/goal',
      '/vigir_move_group/result',
      '/vigir_move_group/status',
      '/pose_body',
      '/move_group/display_planned_path',
      '/hand_controllers/left_hand_traj_controller/follow_joint_trajectory/goal',
      '/hand_controllers/right_hand_traj_controller/follow_joint_trajectory/goal',
      '/hand_controllers/left_hand_traj_controller/command',
      '/hand_controllers/right_hand_traj_controller/command',
      '/vigir_foobar'
   ] </rosparam>
</node>
</launch>
